BE/OwnershipAndControl/Executives.rdf

```diff
@@ -225,9 +230,37 @@
 		<skos:definition>authority to act in a fiduciary capacity with respect to the organization, including but not limited to determining and executing corporate policy</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-be-oac-exec;BoardCompositionControl">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-oac-ctl;Control"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-oac-ctl;hasPartyInControl"/>
+				<owl:onClass>
+					<owl:Class>
+						<owl:unionOf rdf:parseType="Collection">
+							<rdf:Description rdf:about="&fibo-be-oac-cctl;SignificantShareholder">
+							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-be-oac-opty;EntityOwner">
+							</rdf:Description>
+						</owl:unionOf>
+					</owl:Class>
+				</owl:onClass>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-oac-ctl;isInControlOfThing"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-oac-exec;BoardMember"/>
```

Comment:
Maybe this should be BoardMembership, especially given the "period of time" in the definition